packages/suite-web-landing/components/Layout/index.tsx

```diff
@@ -4,7 +4,7 @@ import Translation from '../Translation';
 import { TrezorLogo, Button, colors, variables, Link } from '@trezor/components';
 
 const Layout = styled.div`
-    max-width: ${variables.SCREEN_SIZE.XL};
+    max-width: calc(${variables.SCREEN_SIZE.XL} + 80px);
```

Comment:
a) it's not related to the deprecation banner, right? would be nice to have it separately (at least separate commit, even better separate PR) but let's not be so strict, just keep in mind for next time.

b) It makes no sense to me keeping it related to SCREEN_SIZE.XL if it's not...  What about:
```suggestion
    max-width: 1280px;
```
?

Reply:
there's another layout width used in landing page vs figma design, because of that some texts were differently aligned and we've decided with brand team to change it due figma. I agree it could be another commit.